core/src/main/scala/kafka/log/LogSegment.scala

```diff
@@ -70,21 +77,30 @@ class LogSegment(val log: FileMessageSet,
    *
    * It is assumed this method is being called from within a lock.
    *
-   * @param offset The first offset in the message set.
+   * @param firstOffset The first offset in the message set.
+   * @param lastOffset The last offset in the message set.
+   * @param timestamp The largest timestamp in the message set.
    * @param messages The messages to append.
    */
   @nonthreadsafe
-  def append(offset: Long, messages: ByteBufferMessageSet) {
+  def append(firstOffset: Long, lastOffset: Long, timestamp: Long, messages: ByteBufferMessageSet): Unit = {
```

Comment:
I added that according to the Scala coding style guide. Maybe we can do a refactoring separately if we want to.
http://docs.scala-lang.org/style/declarations.html#methods